index.html

```diff
@@ -233,18 +238,65 @@ <h1>About Myself</h1>
                   </div>
                 </div>
               </li>
+              <hr />
               <li>
                 <div class="skill-card">
                   <div class="s-title"><h2>Skills</h2></div>
                   <div class="s-icon">
                     <i class="fa-sharp fa-solid fa-angle-right"></i>
                   </div>
                 </div>
+                <hr />
               </li>
             </ul>
           </div>
         </div>
       </div>
     </section>
+
+    <!-- Contact Section -->
+
+    <section class="contact">
+      <div class="conatainer grid">
+        <div class="flex">
+          <div class="contact-title">
+            <h2>About Myself</h2>
+          </div>
+          <div class="contact-text">
+            <p>
+              I'm a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don't
+              hesitate to contact me.
+            </p>
+          </div>
+        </div>
+        <div class="flex">
+          <div class="contact-form">
+            <form action="https://formspree.io/f/xbjezjgw" method="POST">
+              <input
+                type="text"
+                name="Name"
+                maxlength="30"
+                placeholder="Your Name"
+                required
+              />
+              <input
+                type="email"
+                name="Email"
+                placeholder="Your Email"
+                required
+              />
+              <textarea
+                name="Message"
+                maxlength="500"
+                placeholder="Write Your Message here"
+              ></textarea>
+              <button class="simple-button" type="submit">Get in touch</button>
+            </form>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
Please consider adding this horizontal element at the end of the section to make your design look closer to the template.

## Figma design:-

![image](https://user-images.githubusercontent.com/94223367/228500864-62add76e-174a-4eb8-b067-325aa9e1c6b3.png)